src/overlays/actors/ovl_En_Bom_Bowl_Man/z_en_bom_bowl_man.h

```diff
@@ -49,8 +49,7 @@ typedef struct EnBomBowlMan {
     /* 0x2D2 */ s16 csId2;
     /* 0x2D4 */ s16 unk_2D4;
     /* 0x2D6 */ s16 csId3;
-    /* 0x2D8 */ struct EnBomBowlMan* unk_2D8[5];
-    /* 0x2EC */ UNK_TYPE1 unk2EC[0x4];
+    /* 0x2D8 */ struct EnBomBowlMan* unk_2D8[6];
```

Comment:
```suggestion
    /* 0x2D8 */ struct EnBomBowlMan* unk_2D8[6]; // The array is 1 indexed with the 0th element being the red shirt bomber
```
Could add the comment here too keep the knowledge as well